src/css/main.css

```diff
@@ -0,0 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;600&display=swap');
+
+:root {
+  --black: #002;
+  --white: #fff;
+  --light-gray: #d3d3d3;
+  --dark-gray: #f3f0f0;
+}
+
+html {
+  overflow-x: hidden;
```

Comment:
Try to fix your `footer` issue and don't use this to conceal it.

Reply:
the ```
overflow:hidden
``` on the html element It's used to hide my mobile menu

Cause I use relative position on my body element and absolute to my nav links and it's giving it a horizontal scroll